implementation/servlet/src/main/java/net/opentsdb/servlet/resources/QueryRpc.java

```diff
@@ -1319,12 +1338,12 @@ static final public RateOptions parseRateOptions(final boolean rate,
 //    return query;
 //  }
   
-  /** @param collector Populates the collector with statistics */
-  public void collectStats(final StatsCollector collector) {
-    collector.record("http.query.invalid_requests", query_invalid);
-    collector.record("http.query.exceptions", query_exceptions);
-    collector.record("http.query.success", query_success);
-  }
+//  /** @param collector Populates the collector with statistics */
```

Comment:
Did you mean to remove this?

Reply:
yup, will clean it up better later on though.